### ISSUE TITLE
Auto convert kebab-case to camelCase

### ISSUE THREAD

**Issue by gilamran** [Quramy/typed-css-modules#12]
css module accepts kebab-case classes, and converts them to camelCase classes, for the same reason that javascript does not accept kebab-case variable.
You can, and should use: [https://www.npmjs.com/package/camelcase](https://www.npmjs.com/package/camelcase)

My suggestion is to auto-convert kebab-case classes to camelCase variables in the d.ts

for example:

```
left-panel {
  width: 30%
}
```

Should be converted to:

```
<SideBar className={css.leftPanel}/>
```

**Comment by Quramy** [comment]
Hi. typed-css-modules only generates `.css.d.ts` files. It doesn't convert CSS modules .css files.
If you want camelized token, I think you should also convert token in your .css at bundling your module.

```javascript
/* the module generated by module bundler plugin(such as css-loader) */
module.exports = {
   "left-panel": "xxx_yyy_zzz_left-panel", /* It's not compatible the .css.d.ts that has camelized token*/
   "leftPanel": "xxx_yyy_zzz_leftPanel", /* who does convert it? */
};
```
I'm not familiar with css-loader or css-modulesify 's option. 
Do you know some options of module bundler plugin to covert camel-case from kebab-case ?

**Comment by gilamran** [comment]
You can pass the query parameter camelCase to the loader: https://github.com/webpack/css-loader#camel-case

**Comment by Quramy** [comment]
Thanks for your reply, it's great! Ok I'll implement this feature.

**Comment by Quramy** [comment]
@gilamran. I've implemented this feature and release it(v0.1.13).
To camelize you set `-c` or `--camelCase` option. Please check out it!
